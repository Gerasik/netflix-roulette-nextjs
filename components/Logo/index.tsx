import React, { FunctionComponent } from 'react';

import styles from './style.module.scss';

const Logo: FunctionComponent = () => {
  return (
    <h1 className={styles.logo}>
      <strong>netflix</strong>
      roulette
    </h1>
  );
};

export default Logo;
