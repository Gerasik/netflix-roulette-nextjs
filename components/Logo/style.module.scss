@import '../../styles/colors.scss';

.logo {
  color: $secondaryFontColor;
  font-weight: normal;
  font-size: 1.5em;
  align-self: flex-start;
  margin: 0.7em 1.2em;
}
