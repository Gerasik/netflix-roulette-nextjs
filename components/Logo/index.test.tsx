import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Logo from './index';

it('Logo', () => {
  const DateInputComponent = renderer.create(<Logo />).toJSON();
  expect(DateInputComponent).toMatchSnapshot();
});

describe('<Footer />', () => {
  it('renders an `h1`', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper.find('h1').length).toBe(1);
  });

  it('renders an `h1`', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper.is('h1')).toBeTruthy();
  });
});
