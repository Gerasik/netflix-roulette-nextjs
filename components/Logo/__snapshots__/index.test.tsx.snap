// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<h1
  className="logo"
>
  <strong>
    netflix
  </strong>
  roulette
</h1>
`;
